### ISSUE TITLE
Fails to start, confused by IPv6 DNS

### ISSUE THREAD

**Issue by vi** [juha-h/baresip-studio#42]
```
07-24 01:49:01.344  4091  4091 W Baresip : zrtp_hash yes
07-24 01:49:01.344  4091  4091 W Baresip : call_volume 0
07-24 01:49:01.344  4091  4091 W Baresip : dns_server fe80::1:53
07-24 01:49:01.344  4091  4091 W Baresip : dns_server 192.168.100.1:53
07-24 01:49:01.344  4091  4091 W Baresip : '
07-24 01:49:01.355  4091  4110 D Baresip Lib: baresip_init() failed (22)
07-24 01:49:01.355  4091  4111 D Baresip Lib: net: dns_server: could not decode `fe80::1:53' (Invalid argument)
07-24 01:49:01.355  4091  4110 D Baresip Lib: stopping UAs due to error: (22)
07-24 01:49:01.355  4091  4111 D Baresip Lib: ua: network init failed: Invalid argument
07-24 01:49:01.355  4091  4110 D Baresip Lib: closing ...
07-24 01:49:01.355  4091  4111 D Baresip Lib: ua: stop all (forced=1)
07-24 01:49:01.355  4091  4110 D Baresip Lib: unloading modules ...
07-24 01:49:01.355  4091  4110 D Baresip Lib: tell main that baresip has stopped
```

**Comment by juha-h** [comment]
Sorry, I haven't checked the issues due to our very short summer.  If fe80::1:53 is a valid IPv6 address, it may be a bug in libre https://github.com/creytiv/re.

**Comment by juha-h** [comment]
There was a bug in checking of DNS Servers that is now fixed in version 8.5.2. If you configure DNS Servers as fe80::1, port 53 is added automatically and DNS Servers becomes  [fe80::1]:53.  With that DNS Servers value, baresip starts OK.

Reopen if you still have the issue after upgrade to 8.5.2.
